pkg/internal/checkup/vmi/spec.go

```diff
@@ -76,6 +76,12 @@ func WithOwnerReference(ownerName, ownerUID string) Option {
 	}
 }
 
+func WithAnnotations(annotations map[string]string) Option {
+	return func(vmi *kvcorev1.VirtualMachineInstance) {
+		vmi.Annotations = annotations
```

Comment:
Please consider the fact that `vmi.WithoutCRIOCPULoadBalancing()`, `vmi.WithoutCRIOCPUQuota()` and `vmi.WithoutCRIOIRQLoadBalancing()` are also touching the annotations.

Reply:
ah you're right